src/appkit.cs

```diff
@@ -6745,6 +6766,11 @@ interface NSFontDescriptor : NSSecureCoding, NSCopying {
 		[Mac (10, 13)]
 		[Export ("requiresFontAssetRequest")]
 		bool RequiresFontAssetRequest { get; }
+
+		[Mac (10,15)]
+		[Export ("fontDescriptorWithDesign:")]
+		[return: NullAllowed]
+		NSFontDescriptor FontDescriptorWithDesign (NSFontDescriptorSystemDesign design);
```

Comment:
Ugh, other APIs are horrible unfortunately, so in the name of constancy we could keep it as is but I think this is better:

```suggestion
		NSFontDescriptor GetFontDescriptor (NSFontDescriptorSystemDesign design);
```

Need other opinions on this.

Reply:
It's a method so it needs a verb. Since it's inside `NSFontDescriptor` and returns a `NSFontDescriptor` then it's best to use `Create` for the verb. Not sure we need anything more (the parameter self-explains the rest)

Sadly most older API did not follow that rule :(

Reply:
I my mind `Create` was reserved for static APIs but I guess that works yes.